components/hab/src/command/pkg/compile.rs

```diff
@@ -12,16 +12,17 @@
 // See the License for the specific language governing permissions and
```

Comment:
It's weird that git thinks this is a rename of components/sup/src/sys/windows/abilities.rs because it certainly is not.